tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -203,6 +203,7 @@ public static int setDateOrder(int dateOrder) {
 
     // Query Configuration
     QUERY_SESSION_TIMEOUT("tajo.query.session.timeout-sec", 60, Validators.min("0")),
+    QUERY_SESSION_CACHE_SIZE("tajo.query.session.cache-size", 1000000, Validators.min("1000000")),
```

Comment:
2. You're right. Validators.min(0) is proper now because we use 0 for cache off option, too.
4. I see. Current implementation does. 
Now, I will update the code soon. Thank you for reviewing, always!